site/assets/adopters/knavigator.yaml

```
@@ -0,0 +1,7 @@
+name: Knavigator
+description: The Knavigator project from NVIDIA uses KWOK to create virtual nodes for testing in kubernetes. 
+contacts:
+- @dmitsh (Dmitry Shmulevich)
```

```suggestion
- dmitsh
```

I think it's a good idea to attach the name.
I'll try to get it from the Github API when generate the page.